### ISSUE TITLE
Implement the work show page

### ISSUE THREAD

**Issue by cam156** [projecthydra/sufia#1065]
Implement the work show page as wireframed.

Dependent on #1002

**Comment by cam156** [comment]
https://wiki.duraspace.org/pages/viewpage.action?pageId=68070466

**Comment by cam156** [comment]
I have broken out choosing a thumbnail for the work from the list of files into a separate ticket #1073.

**Comment by mjgiarlo** [comment]
The work show page is now implemented.

**Comment by scherztc** [comment]
Great job.  Feels real.
:+1:  on Resource Types

**Comment by mjgiarlo** [comment]
Thanks, @scherztc!
